code/game/area/Space Station 13 areas.dm

```diff
@@ -27,15 +22,23 @@ NOTE: there are two lists of areas in the end of this file: centcom and station
 	layer = 10
 	mouse_opacity = 0
 	invisibility = INVISIBILITY_LIGHTING
-	var/lightswitch = 1
-	var/valid_territory = 1 //If it's a valid territory for gangs to claim
 
-	var/blob_allowed = 1 //Does it count for blobs score? By default, all areas count.
+	var/map_name // Set in New(); preserves the name set by the map maker, even if renamed by the Blueprints.
+
+	var/valid_territory = 1 // If it's a valid territory for gangs to claim
+	var/blob_allowed = 1 // Does it count for blobs score? By default, all areas count.
 
 	var/eject = null
 
+	var/fire = null
+	var/atmos = 1
+	var/atmosalm = 0
+	var/poweralm = 1
+	var/party = null

```

Comment:
it's the var that handles the flashing lights in an area
triggered by a fire alarm called PARTY ALARM.